packages/web-components/src/accordion/accordion-item/accordion-item.styles.ts

```diff
@@ -0,0 +1,236 @@
+import { css } from '@microsoft/fast-element';
+import {
+  borderRadiusMedium,
+  borderRadiusSmall,
+  colorNeutralForeground1,
+  colorNeutralForegroundDisabled,
+  colorStrokeFocus1,
+  colorStrokeFocus2,
+  colorTransparentBackground,
+  fontFamilyBase,
+  fontSizeBase200,
+  fontSizeBase300,
+  fontSizeBase400,
+  fontSizeBase500,
+  fontWeightRegular,
+  lineHeightBase200,
+  lineHeightBase300,
+  lineHeightBase400,
+  lineHeightBase500,
+  spacingHorizontalM,
+  spacingHorizontalMNudge,
+  spacingHorizontalS,
+} from '../../theme/design-tokens.js';
+
+export const styles = css`
+  /* --- Default styles --- */
+
+  :host {
+    max-width: fit-content;
+  }
```

Comment:
Do we need a default display property here? Seems like that's missing (using display helper)